src/app/plugins/register.plugin.ts

```diff
@@ -65,9 +66,48 @@ export class RegisterPlugin extends Plugin {
       messageForUser = `Successfully added to ${numSuccessfulClasses} classes`;
     }
 
-    if (invalidClasses.length > 0) {
-      messageForUser += `\nUnable to locate these classes: ${invalidClasses.join(' ')}`;
+    if (invalidClasses.length <= 0) {
+      message.reply(messageForUser);
+      return;
     }
-    message.reply(messageForUser);
+
+    const embededMessage = new MessageEmbed();
+
+    messageForUser +=
+      `\nUnable to locate the following classes: ${invalidClasses.join(' ')}` +
+      `Below you can find suggestions for each incorrect input:`;
+
+    embededMessage.setColor('#0099ff').setTitle('Atleast One Class Not Found');
+    embededMessage.setDescription(messageForUser);
+
+    // I put up to nine so it's there if needed.
+    const emojis: string[] = ['1️⃣', '2️⃣', '3️⃣', '4️⃣', '5️⃣', '6️⃣', '7️⃣', '8️⃣', '9️⃣'];
+
+    const emojiData: IEmojiTable[] = [];
+
+    invalidClasses.forEach((invalidClass: string) => {
+      const curEmote = emojis.shift() || '';
+      const similarClassID =
+        this.container.classService.findSimilarClasses(invalidClass)[0] || 'Nothing Found.';
```

Comment:
Levenshtein seems to be **very** generous on how close you to be to the actual name, but have you thought about what happens when someone tries to register for a class called `Nothing Found`?

Reply:
Would be impossible to do since the input can't have spaces

Reply:
What Im saying is, is that taken care of? Will the bot crash?